### PR TITLE
Payload json to tsvs@0.1.0

### PR DESCRIPTION
**Pull request by edsu7**

### CODE REVIEW

File: payload-json-to-tsvs/README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+```
+
+### Run the package directly
+
+With inputs prepared, you should be able to run the package directly using the following command.
+Please replace the params file with a real one (with all required parameters and input files). Example
+params file(s) can be found in the `tests` folder.
+
+```
+nextflow run icgc-argo/argo-data-submission/download-aspera/main.nf -r download-aspera.v0.1.0 -params-file <your-params-json-file>
```
Comment (lindaxiang) [inline payload-json-to-tsvs/README.md]: download-aspera??

File: payload-json-to-tsvs/main.py [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+        sys.exit('Error: Specified input file %s does not exist or is not accessible!' % args.json_file)
+
+    if not os.path.isdir(args.data_directory):
+        sys.exit('Error: Specified data directory %s does not exist or is not accessible!' % args.data_directory)
+
+    metadata=read_json(args.json_file)
+    make_experiment_tsv(metadata,args.data_directory)
+    make_file_tsv(metadata,args.data_directory)
+    make_read_group_tsv(metadata,args.data_directory)
+
```
Comment (lindaxiang) [inline payload-json-to-tsvs/main.py:56]: `data_directory` is only needed by `make_file_tsv`.

**Review by lindaxiang — APPROVED** [review]

File: payload-json-to-tsvs/main.py (hunk 2 above)
Comment (edsu7) [reply payload-json-to-tsvs/main.py:56]: Fixed.

File: payload-json-to-tsvs/README.md (hunk 1 above)
Comment (edsu7) [reply payload-json-to-tsvs/README.md]: Fixed:
https://github.com/icgc-argo/argo-data-submission/blob/e39c1728161f2b32327f9885b78a638bc1304c28/payload-json-to-tsvs/README.md?plain=1#L39

**Review by lindaxiang — APPROVED** [review]

**Pull request merged**